conf.d/main

```diff
@@ -18,13 +18,22 @@ CONF=$WEBROOT/.env
 LOC_SHARE=/usr/local/share
 LOC_BIN=/usr/local/bin
 
+#Update php related config files for ioncube loader
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/apache2/php.ini
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/cli/php.ini
```

Comment:
Something Stefan and I have started doing recently is instead of using the specific php version in the path to php.ini files (i.e. `8.2` in the above 2 lines), instead use `?.?`. In [shell globbing](https://tldp.org/LDP/abs/html/globbingref.html) `?` matches any single character. So `?.?` will match any PHP version. I.e. instead of this:
```
/etc/php/8.2/cli/php.ini
```
use this:
```
/etc/php/?.?/cli/php.ini
```
I'd probably put the unpacking and clean up lines (i.e. the lines I noted above) above these sed lines - so other than the initial download, all the ioncube stuff is here together.

Reply:
I substituted the php version to the wildcard characters. I also made sure the ioncube install is in the order you suggested within the conf.d/main file.